### ISSUE TITLE
Renderer constructor changes active framebuffer binding

### ISSUE THREAD

**Issue by toolness** [servo/pathfinder#215]
While working on the Unity plugin, I noticed that the Renderer constructor appears to change the framebuffer binding (in OpenGL at least) without changing it back to whatever it originally was once it is done.  I currently do this myself in the plugin but I wasn’t sure if you’d consider it a bug or feature so I am reporting it here just in case.

**Comment by toolness** [comment]
Oh weird, I submitted this issue on my phone so it looks like it somehow got added twice. This is a duplicate of #214.
